core/sdk/src/extn/client/extn_client.rs

```diff
@@ -285,6 +285,14 @@ impl ExtnClient {
                                 let mut ripple_context = self.ripple_context.write().unwrap();
                                 ripple_context.deep_copy(context);
                             }
+                            let is_ripple_context =
+                                RippleContext::is_ripple_context(&message.payload).is_none();
+
+                            if self.has_event_listener(&message.target.as_clear_string()) {
+                                continue;
+                            } else if !is_ripple_context {
+                                warn!("No valid processors for the event {:?}", message);
+                            }
```

Comment:
I'm not following the purpose of the `is_ripple_context` check, the ask in the ticket is unclear to me. @satlead can you clarify the ask there?

Reply:
This was an incorrect warning and there shouldnt be one for Extensions which do not have a registered Ripple Context processor. This warning is intended for messages which is not ripple context and sent to an extension without a processor

Reply:
Will we not already display a similar error when `handle_vec_stream()` is called? I'm trying to understand why it's necessary here.